src/components/Comments/ConnectorWrapper.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {View, StyleSheet} from 'react-native';
 
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the `COLORS` object from the `theme` module instead of the `colors` object from the `colors` module. This change is consistent with the PR's goal of standardizing color usage across the application. However, it's important to ensure that the `COLORS` object contains all the necessary color values and that these values are correctly mapped to their corresponding keys.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->